OOP.Task4/Program.cs

```diff
@@ -5,13 +5,13 @@ namespace OOP.Task4
 {
     class Program
     {
-        static string Input { get; set; }
+      public static string Input { get; set; }
 
            Program()
```

Comment:
please, set constructor access modifier.
and could you please fix the alignment?